### ISSUE TITLE
Password

### ISSUE THREAD

**Issue by GilBillieri** [sepinf-inc/IPED#1687]
How should the password of a compressed file be entered on the command line so that Iped has the ability to process it correctly?
As described in the help command is possible through -p, -password (password for encrypted images/volumes). 
Or not possible for .zip files or even Files originated from an iTunes backup?

**Comment by lfcnassif** [comment]
Today -p password supports just APFS file system decryption.

Implementing it for single zip file input should not be hard. But for several different zip files with different passwords inside a folder or inside a forensic image is not straightforward. Which is your use case exactly?

**Comment by GilBillieri** [comment]
We did that data extraction from iTunes with the encryption option enabled; How can I pass the password parameter in the processing? Physical Analyzer asks for the password when ordering to process iOS backup, IPED also allows this ?

**Comment by lfcnassif** [comment]
No, currently it's not supported. Even if it was, I wouldn't recommend processing an iTunes backup directly by IPED because we have very few iOS artifacts parsers today.

**Comment by lfcnassif** [comment]
I'm moving to discussions.
